ConsoleApp/Food.cs

```diff
@@ -0,0 +1,23 @@
+namespace ConsoleApp;
+    class Food
+    {
+        int x;
+        int y;
+        char ch;
+        public Point food { get; private set; }

```

Comment:
code convention violated, should be
```suggestion
        public Point Food { get; private set; }

```